e2e/collections_test/test1/test.yml

```diff
@@ -0,0 +1,6 @@
+query: > 
+  CALL collections.min([1,2,3,4,5]) YIELD min RETURN min;
+
+output:
+  - min: 1
+  
```

Comment:
You somehow have empty line and then there is this red circle :joy: try deleting one line and then it should be okay, but press enter after `- min: 1` line